### DISCUSSION TITLE
del button and annotation box deleted

### DISCUSSION THREAD

**Discussion by qwest0** [sumatrapdfreader/sumatrapdf#4191] [Q&A]
accidently pressed del button and annotation box deleted , how to undo or find the typed info ,please help

**Comment by GitHubRulesOK** [comment]
@qwest0 
If there was no save before delete the data is likely lost. 

It is not a good idea to keep saving after every small addition
but it is a good idea to save after an hour (or half hours) work of many changes
so they are flushed into the end of the file.

**Reply by qwest0** [reply]
i dont save it again and again , its when i type and i find some mistyping , i select or move to the word or letter and i press del button as a habit - del button should delete the words or letters but instead in sumatra - it deletes the whole annotation box -- at this immediate time i would like to somehow find the typed info from the deleted annotated box ,so this was my query and also a request to change the function of del button in future versions .

**Comment by kjk** [comment]
This was a bug and has been fixed in pre-release https://www.sumatrapdfreader.org/prerelease

**Reply by qwest0** [reply]
oh yes it works now as it should with del button , wish i knew this before , so many times i have deleted my annotations while typing by pressing del button ! thank for this pre release. anything i should watch out in this pre release ? i mean any bugs or any messed up things ?

**Reply by kjk** [reply]
No. If I know about a major bug, I fix it.
